orders/serializers.py

```diff
@@ -3,8 +3,16 @@
 
 
 class OrderSerializer(serializers.ModelSerializer):
-    status = serializers.IntegerField(required=False)
 
     class Meta:
         model = Order
-        fields = '__all__'
+        fields = ('requester', 'grantor', 'order_price', 'description')
+
+    def validate(self, attrs):
+        requester = attrs['requester']
+        order_price = attrs['order_price']
+
+        requester_account_balance = requester.account.balance
```

Comment:
Muda o atributo só pra balance. Pelo contexto dá pra entender.

Reply:
Vou manter porque esse método de validação utiliza todos os campos. Então podemos em determinado momento trabalhar com o balance do `grantor`.  E seguindo também o zen do Python: Explicit is better than implicit.

https://www.python.org/dev/peps/pep-0020/

Reply:
Não dá pra criar um método pra cada validação?
Um salsichão vai ficar chato de ler depois '-'.

Reply:
`Explicit is better than implicit` não tem nada de implicito aqui. Mas ok xD

Reply:
Dá sim! Acho que à medida que formos implementando a gente vai desacoplando.